sklearn/gaussian_process/kernels.py

```diff
@@ -92,6 +92,13 @@ def __new__(cls, name, value_type, bounds, n_elements=1, fixed=None):
         return super(Hyperparameter, cls).__new__(
             cls, name, value_type, bounds, n_elements, fixed)
 
+    def __eq__(self, other):
```

Comment:
This is not useful internally. I just added it for testing purposes. (So that I can test that the hyperparameters of the parent and cloned kernel are the same directly)